span-normalizer-api/src/main/avro/trace-identity/TraceIdentity.avdl

```diff
@@ -0,0 +1,7 @@
+@namespace("org.hypertrace.core.spannormalizer")
+protocol TraceIdentityProtocol {
+  record TraceIdentity {
+    union {null, string} tenant_id = null;
```

Comment:
tenant_id can never be null. It should either come in the span data or default tenant id should be enabled.

Reply:
seconded